config/initializers/okcomputer.rb

```diff
@@ -18,6 +18,15 @@ def version
 end
 OkComputer::Registry.register 'version', VersionCheck.new
 
+# Pings a specific solr core -- assumes Solr is configured to use PingRequestHandler
+# @see https://cwiki.apache.org/confluence/display/solr/Ping
+class SolrCoreCheck < OkComputer::HttpCheck
```

Comment:
What's the difference between this and https://github.com/sportngin/okcomputer/blob/master/lib/ok_computer/built_in_checks/solr_check.rb?